src/scss/components/checkbox/_index.scss

```diff
@@ -179,7 +179,7 @@ hx-checkbox-control > input[type="checkbox"]:disabled {
   }
 
   ~ label {
-    color: $gray-700;
+    color: var(--hxCheckbox-disabled-label, $gray-700);
```

Comment:
please change to var(--hxCheckbox-disabled-label-color, $gray-700);

Reply:
Changes have been done, Please review